yeezyCopper.js

```diff
@@ -60,69 +64,153 @@ function pay () {
     }
   });
 
-  $('input#dwfrm_payment_creditCard_cvn').val(function () {
+  $('input#dwfrm_payment_creditCard_cvn').val(function() {
     return config.security_code;
   });
 
-  setTimeout(function () {
+  setTimeout(function() {
     placeOrder();
   }, 300);
 }
+
+function _x(STR_XPATH) {
+  var xresult = document.evaluate(STR_XPATH, document, null, XPathResult.ANY_TYPE, null);
+  var xnodes = [];
+  var xres;
+  while (xres = xresult.iterateNext()) {
+    xnodes.push(xres);
+  }
+
+  return xnodes;
 }
 
-function addNewShipping() {
-  $('.textinput.firstname').val(function () {
-    return config.first_name;
-  });
+function sleep(milliseconds) {
+  var start = new Date().getTime();
+  for (var i = 0; i < 1e7; i++) {
+    if ((new Date().getTime() - start) > milliseconds) {
+      break;
+    }
+  }
+}
 
-  $('.textinput.lastname').val(function () {
-    return config.last_name;
-  });
 
-  $('.textinput.address1').val(function () {
-    return config.street_address_1;
-  });
+function checkovernightShip() 
+{
```

Comment:
I didn't even know what K & R style was lol thank you for pointing that out!! 
I think I fixed it, does this look right?
<img width="551" alt="screen shot 2016-06-15 at 9 23 48 pm" src="https://cloud.githubusercontent.com/assets/5168320/16102646/7d1f389e-333f-11e6-8db1-a82560dc73f1.png">